airbyte-webapp/src/components/ui/DropdownMenu/DropdownMenu.module.scss

```diff
@@ -9,11 +9,10 @@
 .items {
   z-index: z-indices.$dropdownMenu;
   overflow: auto;
-  width: 260px;
   max-height: 300px;
-  padding: variables.$spacing-sm 0;
+  padding: variables.$spacing-sm;
   outline: none;
-  border-radius: variables.$border-radius-xs;
+  border-radius: variables.$border-radius-md;
```

Comment:
These changes make this DropdownMenu look a bit more consistent with the style of other airbyte components